### ISSUE TITLE
BP alerts retry at the same time

### ISSUE THREAD

**Issue by Asiramb** [sni/Thruk#447]
When there is a BP alert, the first and second attempts are at the same time, so the retry interval is not working in that case. However, the third attempt is done one minute after.

Service Ok[2015-02-16 06:11:05] SERVICE ALERT: Dali;Dali;OK;SOFT;3;OK - everything is fine
Service Critical[2015-02-16 06:10:05] SERVICE ALERT: Dali;Dali;CRITICAL;SOFT;2;CRITICAL - Dali_webinject failed
Service Critical[2015-02-16 06:10:05] SERVICE ALERT: Dali;Dali;CRITICAL;SOFT;1;CRITICAL - Dali_webinject failed

Is it a bug? In other case, where can I change the max_attempts for BP?
Thanks.

**Comment by sni** [comment]
max_check_attempts can be changed by using different templates or changing the default template.
Besides that, i cannot reproduce this issue, is this still the case?

**Comment by Asiramb** [comment]
Thanks for your answer. I didn't see that BP template was the default one ;-)
It still happens, but I have changed the default template, as you mentioned, in order to do three attempts (in configuration max_check_attemps is set to 4 although). For me it is OK.

To confirm de issue, I visited the demo site http://demo.thruk.org/thruk/cgi-bin/login.cgi and if you see the alert history for a BP, the SOFT1 and SOFT2 alerts have the same timestamp.
